tasks/projects.yml

```diff
@@ -215,10 +215,10 @@
       keypair: "{{ item.1.name }}"
 
 - name: Ensure quotas are set
-  os_quota:
+  openstack.cloud.quota:
```

Comment:
Why do we need to use the FQCN here? Isn't there a redirect from os_quota to quota?

Reply:
I thought it would be good to mark explicitly that the collection is in use and not to rely on the redirect. It seems to be the recommended approach [[1]](https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_2.10.html#ansible-2-10-porting-guide) [[2]](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#simplifying-module-names-with-the-collections-keyword).

Reply:
We have talked about moving to FQCNs in the roles, but I'd rather see a consistent approach applied across all of the roles, and especially within a single role.

We need to be careful about when to apply the change, since it would break older versions of ansible which may still be in use. I suggested that moving the roles into a stackhpc.openstack collection could be a good point at which to make the change.

In the mean time, you can still use the collection with the existing code, right?

Reply:
I've been confused by incorrect `os_quota` documentation - see https://github.com/stackhpc/ansible-role-os-projects/issues/36; it looked like you'd need a module from the openstack.cloud collection to have better support for volume quotas, seems not.